blog/README.md

```diff
@@ -0,0 +1,83 @@
+
+<p align="center"><img alt="Captain America sitting on a backwards chair" src="https://res.cloudinary.com/cloudfour/image/upload/c_scale,f_auto,q_auto,w_800/v1580506044/america_obfr70.jpg" width="400" /></p>
+
+So you've decided to write for the Cloud Four blog… Congratulations! There are a few things you may want to know about how our blog works:
+
+## Post Excerpt Field
+
+Don't miss the "Excerpt" field at the bottom of the post editing UI. By crafting a short excerpt here, it will be displayed on the post listing pages and used for social media posts. If you don't set one explicitly, one will be created using the first few sentences of your post.
+
+## Featured Image Field
+
+Similarly, don't forget the "Featured Image" field in the sidebar. If you set an image here, it will be used for social media. This image is not displayed in the post content, however, so if you want to use it as a big image at the top of your post, you'll have to do that AND set the featured image.
+
+## WordPress Shortcodes & Classes
+
+We have a small collection of [useful shortcodes and classes](patterns.md) to apply design patterns from our [pattern library](https://cloudfour-patterns.netlify.com/patterns/utilities.html).
+
+## Previewing
+
+Note that our blog has two preview features. The "Preview" button only works for authenticated users. The "Enable Public Preview" checkbox will give you a public URL to share with unauthenticated users.
+
+Sometimes the public preview can get stuck or cached. To reset, just uncheck the preview box and check it again.
+
+## Publication Time
+
+Jason's favorite time is 8:30 AM Pacific Time. Before lunch on East Coast, but late enough to be what people on West Coast see it first thing when they arrive in office.
+
+## Comments & Pingbacks
+
+We don’t have a formal policy on comment approval, but here’s Jason's general guideline:
+
+- Let the author approve comments so they have the opportunity to process and respond if they want to.
+- Replying to comments is up to the author. If they contribute to understanding and there is nothing else to add, don’t feel compelled to respond.
+- Delete comments that are spam, troll, or jerk-like behavior.
+
+### Advice from Jason:
+
+> Despite the fact that we ask people to be kind, courteous and constructive when they submit comments on our site, when articles reach a larger audience, we will inevitably end up some with comments that start to inch into an unkind territory. It can be hard to tell how to respond. If they are blatant jerks, I’ll simply delete their comment—you don’t get to come into our house and tear the place up.
+
+> Sometimes with a comment [like this one](https://cloudfour.com/thinks/an-html-attribute-potentially-worth-4-4m-to-chipotle/#comment-5880), I’ll decide to ignore the sarcasm and treat the question seriously. [Here's another example](https://cloudfour.com/thinks/first-thing-you-should-do-to-optimize-your-desktop-site-for-mobile/#comment-690).
```

Comment:
While I dig this quote, since this is a public repo I'm not sure we want to put specific commenters on blast? Is there a way we could harvest the advice from these quotes in a more generic way?

Reply:
That's fair. I'll rewrite it to be more generic "try responding to sarcasm with earnest information" kinda thing.